projects/serializers/project.py

```diff
@@ -106,6 +111,32 @@ def _get_users(project, list_view=False):
 
         return UserSerializer(users, many=True).data
 
+    @staticmethod
+    def _get_updates(project):
+        # Get the latest attribute updates for distinct attributes
+        actions = (
+            project.target_actions.filter(verb="updated attribute")
```

Comment:
Hmm, maybe we could have `updated attribute` in a variable/constant so it doesn't have to be hard-coded in two places.

Reply:
Good point! Will change!